contrib/widget-examples/lemonbar/lib.lua

```diff
@@ -0,0 +1,34 @@
+function colorize (text, fg, bg, u)
+  text = ( fg and '%{F' .. fg .. '}' .. text .. '%{F-}' ) or text
+  text = ( bg and '%{B' .. bg .. '}' .. text .. '%{B-}' ) or text
+  text = ( u and '%{U' .. u .. '}' .. text .. '%{U-}' ) or text
+  return text
+end
+
+function pager (status)
+  local colors = {
```

Comment:
This thing should not be local. In fact, `lib.lua` should be imported with `requre`, not with `dofile.